site/content/en/blog/releases/1.20.0-rc.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "1.20.0-rc - simple game server version to 0.6"
```

Comment:
This line should be `title: "1.20.0 - Release Candidate"`

Reply:
Good catch, I missed that one!